### ISSUE TITLE
Table capabilities

### ISSUE THREAD

**Issue by romankolosok** [pdfme/pdfme#484]
### Is your feature request related to a problem? Please describe.

I want to have capability to use different fonts within a table, for example I need to create a table, where some fields are just text, and other fields are barcodes to that item in that row. I want to be able to change font in the barcode cell to an actual barcode39/128 font.

### Describe the solution you'd like

I would like to be able to have an option to select a custom font for each cell in the table, to be able to generate auto table where each row contains text of different fonts.

### Describe alternatives you've considered

I considered dynamically creating however many barcodes and putting them near the corresponding table cell but encountered issue with dynamic table, if my text is longer than the column width, it wraps around and increases height of the row and I cannot determine height of the row before rendering the pdf, so my barcodes are not mapping to the rows they are corresponding to.

**Comment by hand-dot** [comment]
@romankolosok 
Are you saying that you want to place barcodes within table cells?

**Comment by romankolosok** [comment]
Yes, I had to work around and "draw" table manually with rectangles and static height to put barcodes in it. And it significantly affects pdf generation time I'm pretty sure, I think it would be very helpful If I could embed barcode within a table cell.

**Comment by romankolosok** [comment]
I tested it on the biggest data feed I had: around 900 rows of data, It took around 4 minutes to generate

**Comment by hand-dot** [comment]
I'd like to treat performance issues as a separate problem for now. While I understand that the nested for loops are degrading performance, there are still bugs in the table itself.

I think it's a good suggestion to allow specifying schemas other than text in cells, but we're not at a stage where we can incorporate that suggestion yet. However, I'm not rejecting this proposal, so I'd like to change the title to something more appropriate.

We might be able to implement this once the table bugs and performance issues are resolved.

**Comment by brian-scb** [comment]
👍 also looking for similar. Nested tables would be awesome.

**Comment by joernroeder** [comment]
images or custom schemas inside of cells would be awesome too

**Comment by nicofishman** [comment]
Yeah, it would be awesome to be able to put checkboxes inside of table cells

**Comment by hand-dot** [comment]
ref: https://github.com/pdfme/pdfme/pull/1241/files

**Comment by superiority-claw-agent** [comment]
PDFweave's plugin `measure` hook + smart tables let cells host arbitrary custom-rendered content (barcodes, images, etc.) by registering a custom plugin and using `binding.columns` to point cells at the data. See @pdfweave/schemas tables + [docs/examples/dynamic-height-custom-schema.md](https://github.com/IDNTEQ/pdfweave/blob/main/docs/examples/dynamic-height-custom-schema.md).

If a maintainer wants this surfaced upstream too, happy to send a PR — but the work is already in PDFweave at v0.1.0. Context on the fork: pdfme/pdfme#1468.

— filed by @superiority-claw-agent on behalf of the PDFweave maintainers (IDNTEQ)

**Comment by hand-dot** [comment]
Linked #1055, which adds row/cell-level style control and is relevant to the per-cell font customization described here. It does not add embedded barcode/image schema objects or row sizing around those objects. The broader non-text table-cell request remains open.

**Comment by hand-dot** [comment]
Maintainer triage note (current `main`): `still_missing`.

Rationale: tables remain string-grid based (`head: string[]`, body/content as `string[][]`). Cell rendering still delegates to the internal text-cell renderer with string values. There is no cell type/API for barcode, image, nested table, checkbox, or custom schema rendering inside a table cell. Designer controls are table/head/body/limited column styles, not per-cell plugin binding.

Suggested action: keep open / needs design. This needs a cell content model, measurement hooks, pagination behavior, and Designer editing semantics.

Repro: in the Playground Designer, select a table and inspect its property panel. It exposes text-oriented table/head/body/column controls only; there is no non-text cell binding or custom schema cell renderer. No issue closure or PR action taken.
